### DISCUSSION TITLE
New field for staff members hidden from API.

### DISCUSSION THREAD

**Discussion by Picsou993** [HypixelDev/PublicAPI#465] [Ideas]
Currently, when a staff member hides itself from the api, we only have few information and that's intended.
However would it be possible to have a field `"hidefromapi":true` when they are hidden from it so we can detect it more easily.

**Comment by mdashlw** [comment]
This would be helpful, but unfortunately it was already suggested a long time ago, and since it still hasn't been implemented, I assume it was rejected
